### PR TITLE
Issue Fixed : Hero Section

### PR DESCRIPTION
**Pull request by Akash-moon**

### CODE REVIEW

**Comment by AbhiPatel10** [comment]
Add Details and Screenshot when you create PR. also add " close # issue name"  so it will automatically close that issue which you have generated PR for

File: index.html [hunk 1]
```diff
@@ -121,6 +121,70 @@
...
+                    <svg id="filters">
+                        <defs>
+                            <filter id="threshold">
+                                <feColorMatrix in="SourceGraphic" type="matrix" values="1 0 0 0 0
+                                                        0 1 0 0 0
+                                                        0 0 1 0 0
+                                                        0 0 0 255 -140" />
+                            </filter>
+                        </defs>
+                    </svg>
```
Comment (AbhiPatel10) [inline index.html:170]: What is this for ?

File: index.html [hunk 2]
```diff
@@ -121,6 +121,70 @@
...
+                    <a href="#">Blog</a>
+                    <a href="#">Shop</a>
+                    <a href="#">Contact</a>
+                </div>
+                <label for="check">
+                    <i class="fas fa-bars menu-btn"></i>
+                    <i class="fas fa-times close-btn"></i>
+                </label>
+            </header> -->
+
```
Comment (AbhiPatel10) [inline index.html:150]: Remove unwanted code

File: js/main.js [hunk 3]
```diff
@@ -91,3 +91,87 @@ function fun() {
...
+          textIndex++;
+      }
+
+      doMorph();
+  } else {
+      doCooldown();
+  }
+}
+
+animate();
```
Comment (AbhiPatel10) [inline js/main.js:177]: Write code your self do not copy from Internet.

File: img/bg/herobackground.jpg
Comment (AbhiPatel10) [inline img/bg/herobackground.jpg:1]: Blur and add some darkness on photo so text can be seen easily.

File: index.html (hunk 1 above)
Comment (Akash-moon) [reply index.html:170]: This code if for animation

File: index.html (hunk 2 above)
Comment (Akash-moon) [reply index.html:150]: Done

File: js/main.js (hunk 3 above)
Comment (Akash-moon) [reply js/main.js:177]: Actually I was having a little trouble to understanding the animation part , so I just use the template from the internet 
and rest of the work I do it by myself .

Comment (AbhiPatel10) [reply js/main.js:177]: That's why you facing the issue in responsive

**Comment by AbhiPatel10** [comment]
image

Remove Horizontal Scrolling

**Comment by AbhiPatel10** [comment]
I Suggest you to write whole code again do not copy paste directly from internate understand first how the code is working

**Pull request closed without merging**